network-api/networkapi/wagtailpages/__init__.py

```diff
@@ -16,3 +20,54 @@ def register_howto_menu_item():
         _('How Do I Wagtail'), reverse('how-do-i-wagtail'),
         name='howdoIwagtail', classnames='icon icon-help', order=900
     )
+
+
+# WARNING: this is not necessarily a good idea, but is the only way to override
+# Django's default behaviour of requiring language codes to be lowercased.
+# We have to modify the core Django method, because we have no way to replace
+# all the core functionality that relies on this - e.g., url resolvers that
+# the Django admin and third party apps use.
+# This is pretty ugly, and ideally this should be fixed upstream.
+
+
+def language_code_to_iso_3166(language):
+    """Turn a language name (en-us) into an ISO 3166 format (en-US)."""
+    language, _, country = language.lower().partition('-')
+    if country:
+        return language + '-' + country.upper()
```

Comment:
let's use formatted strings here. It's a minor change, but avoiding string concatenation in favour of string templating is always a good idea

```suggestion
        return f'{language}-{country.upper()}'
```

Reply:
Thanks, do you want me to update this on donate too?